extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -916,6 +916,11 @@ tags: {  # <amp-story-page-attachment> with no href
     value: "dark"
     value: "light"
   }
+  attrs: {
+    name: "cta-drawer-theme"
```

Comment:
Can we find a name that leaves 0 doubt as to what it styles?

`amp-story-page-attachment[theme]` used to do both
`[cta-drawer-theme]` honestly I don't know if it styles the drawer, the CTA, or the drawer of the CTA (in the case of an `attachment[href]`)

Reply:
It's the drawer that displays inline attachments after they've been tapped.
How about `inline-cta-drawer-theme`?

Reply:
I wonder if `drawer-theme` works. It might have enough context since it's an attribute of `amp-story-page-attachment`.